common/src/main/java/gov/cms/ab2d/common/util/Constants.java

```diff
@@ -83,4 +83,6 @@ private Constants() { }
     public static final String SINCE_EARLIEST_DATE = "2020-02-13T00:00:00.000-05:00";
 
     public static final String ZIPFORMAT = "application/zip";
+
+    public static final String HPMS_ORGANIZATION = "HPMS_AUTH";
```

Comment:
Why is this defined globally?  This goes very much against what we are trying to do which is to break up the monolith.  It should be either defined in the HPMS module or in the EVENT client jar.